gazebo/msgs/msgs.cc

```diff
@@ -1778,6 +1778,16 @@ namespace gazebo
       if (_sdf->HasElement("shadows"))
         result.set_shadows(_sdf->Get<bool>("shadows"));
 
+      if (_sdf->HasElement("shadow_caster_material_name"))
```

Comment:
I think we should use the `ignition:` xml namespace prefix for this element, since we haven't defined it in the SDFormat spec. (see https://github.com/osrf/gazebo/pull/2960/files#diff-01e2cc89eb42d64422714328f474a4d505624ee5187bb6ca666a5dae6db784e4R475)